File: app/src/main/java/com/smart/resources/schools_app/database/dao/StudentAbsenceDao.java
package com.smart.resources.schools_app.database.dao;

import com.smart.resources.schools_app.database.model.ExamModel;
import com.smart.resources.schools_app.database.model.StudentAbsenceModel;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface StudentAbsenceDao {

    @GET("studentAbsence")
    Call<List<StudentAbsenceModel>> fetchstudentAbsence();
}

File: app/src/main/java/com/smart/resources/schools_app/database/dao/StudentAbsenceDao.kt
package com.smart.resources.schools_app.database.dao

import com.smart.resources.schools_app.database.model.StudentAbsenceModel
import retrofit2.Call
import retrofit2.Response
import retrofit2.http.GET

interface StudentAbsenceDao {
    @GET("studentAbsence")
    suspend fun fetchStudentAbsence(): Response<List<StudentAbsenceModel>>
}